<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindowClass</class>
 <widget class="QMainWindow" name="mainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>oldPhotoRestoration</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <attribute name="title">
        <string>Labeling</string>
       </attribute>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>10</y>
          <width>291</width>
          <height>421</height>
         </rect>
        </property>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="geometry">
          <rect>
           <x>19</x>
           <y>9</y>
           <width>251</width>
           <height>36</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="btnBack">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnRedo">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnCompare">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QTabWidget" name="tabColorSpace">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>261</width>
           <height>161</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabRGB">
          <attribute name="title">
           <string>RGB</string>
          </attribute>
          <widget class="QToolButton" name="btnSelectRGB">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>select</string>
           </property>
          </widget>
          <widget class="QToolButton" name="btnCancelRGB">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>20</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>cancel</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tabHSV">
          <attribute name="title">
           <string>HSV</string>
          </attribute>
          <widget class="QWidget" name="widget_6" native="true">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>231</width>
             <height>40</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QSpinBox" name="spinBox"/>
            </item>
            <item>
             <widget class="QSlider" name="hsvSlider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_7" native="true">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>231</width>
             <height>61</height>
            </rect>
           </property>
           <widget class="QToolButton" name="btnSelectHSV">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>10</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>select</string>
            </property>
           </widget>
           <widget class="QToolButton" name="btnCancelHsv">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>10</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>cancel</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>51</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>260</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>290</y>
           <width>181</width>
           <height>36</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QToolButton" name="btnDilate">
            <property name="text">
             <string>dilate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnErode">
            <property name="text">
             <string>erode</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>350</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>370</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="btnReset">
            <property name="text">
             <string>reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnSave">
            <property name="text">
             <string>save</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="ImageView" name="labelingGraphicView">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>10</y>
          <width>491</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Restoring</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSelectRootPath"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionUserConfig"/>
   </widget>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="actionSelectServer"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuServer"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSelectRootPath">
   <property name="text">
    <string>Files</string>
   </property>
   <property name="toolTip">
    <string>Select root path for images</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>save the processed image</string>
   </property>
  </action>
  <action name="actionStorage">
   <property name="text">
    <string>storage</string>
   </property>
  </action>
  <action name="actionSelectServer">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionUserConfig">
   <property name="text">
    <string>User Config</string>
   </property>
   <property name="toolTip">
    <string>Set up user config</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header location="global">E:\Code\CPP\oldPhotoRestoration\oldPhotoRestoration\ImageView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="oldPhotoRestoration.qrc"/>
 </resources>
 <connections/>
</ui>
